test/persistent_receiver_test.go

```diff
@@ -148,6 +149,31 @@ var _ = Describe("PersistentReceiver", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(fmt.Sprint(receiver)).To(ContainSubstring(fmt.Sprintf("%p", receiver)))
 		})
+		Context("with a connected messaging service and specially formatted queue", func() {
+            const basicQueueName = "test_invalid_durability_on_bind_raises_exception"
+            modifiedQueueName := "#P2P/QTMP/v:solbroker/" + basicQueueName
```

Comment:
Does this only work as solbroker is the default hostname? Is there a way to pull the docker hostname?
Does this break again remote broker hostname?

Reply:
I agree with @cjwmorgan-sol. Please is there a way to retrieve the hostname dynamically?